i18n/zh/docusaurus-plugin-content-docs/current/user-guide/administration/manage-metadata/manage-etcd.md

```diff
@@ -0,0 +1,537 @@
+---
+keywords: [etcd, kubernetes, helm, 备份, 恢复]
+description: 管理 etcd 集群的综合指南，包括使用 kubernetes 和 helm 的安装、备份和恢复过程。
+---
+
+# 管理 etcd
+
```

Comment:
Did you forget to delete `i18n/zh/docusaurus-plugin-content-docs/current/user-guide/administration/manage-etcd.md`?